packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -508,7 +508,7 @@ const extractConfigurationsFromInputs = async () => {
 					}
 					if (data?.status === TaskStatus.Success) {
 						logger.success('Model configured from document(s)');
-						const outputJSON = JSON.parse(new TextDecoder().decode(data.output));
+						const outputJSON = JSON.parse(atob(data.output));
 						console.debug('Task success', outputJSON);
```

Comment:
Is this causing the infinite successes? Why are we switching to `atob`? Do we need to handle non-ascii encodings?

Reply:
`data.output` is a base64 string.  I took a quick look at the [TextDecoder documentation](https://developer.mozilla.org/en-US/docs/Web/API/TextDecoder) and it look like it doesnt support base64, Im not sure why we were using it in the first place.  

Anyways, `TextDecoder` it was causing an uncaught error since it couldnt recognize base64 and the `onData` function would be called infinitely since the connection wasn't closing (hence infinite success toasts).  Ive added error handling in the `onData` function to close the connection when we get an error which was the root cause of the infinite toasts.

Reply:
@YohannParis done